config/settings.d/dns_nsupdate.yml.example

```diff
@@ -0,0 +1,16 @@
+---
+#
+# Configuration file for 'nsupdate' dns provider
+#
+# Can be true, false
+:enabled: true
```

Comment:
Noticed one thing - we have dns_provider setting and enabled/disabled for all. Can't we just assume that the only enabled provider is the one user wants to use? And error out if the number of enabled providers is not 1.